src/app/zones/views/ZonesList/ZonesListTable/AddZone/AddZone.test.tsx

```diff
@@ -37,7 +37,7 @@ describe("ZonesListForm", () => {
     render(
       <Provider store={store}>
         <MemoryRouter>
-          <ZonesListForm closeForm={vi.fn()} />
+          <AddZone closeForm={vi.fn()} />
         </MemoryRouter>
       </Provider>
     );
```

Comment:
Totally optional, but we have a util function `renderWithBrowserRouter` that automatically wraps components with the store provider and memory router, we're trying to use it everywhere we can 

```suggestion
    renderWithBrowserRouter(
      <AddZone closeForm={vi.fn()} />,
      { store }
    )
```

Reply:
Done 👍